nautobot/core/templatetags/helpers.py

```diff
@@ -642,6 +642,11 @@ def render_uptime(seconds):
 
 @library.filter()
 @register.filter()
+def dbm(value):
+    """Display value as dBm."""
+    return f"{value} dBm" if value else placeholder(value)
+
+
 def hyperlinked_field(value, hyperlink=None):
```

Comment:
Looks like the `@library.filter()` and `@register.filter()` were unintentionally removed from this function?